dictionaria/models.py

```diff
@@ -127,6 +127,29 @@ def links_to(self):
                 lambda s: s.description):
             yield desc, [a.target for a in assocs]
 
+    @property
+    def relations(self):
```

Comment:
I always try to prefix things that return generators with `iter`. Otherwise it's really easy to get bitten when not iterating over it or trying to iterate twice ...

Reply:
Hm, the two methods right above this one already seem to violate this -- however, a quick `grep` told me that they aren't actually called anywhere (the only call was replaced with the call to `relations`).
Maybe I should just remove those…

Reply:
I kind of expected this - hence the "always **try**" in my comment :) - yes, remove - but also check whether they are used in templates.